app/models/topic/content_tagged_to_topic.rb

```diff
@@ -45,8 +45,16 @@ def search_params
     {
       start: 0,
       count: PAGE_SIZE_TO_GET_EVERYTHING,
-      filter_specialist_sectors: [@topic_slug],
+      filter_name => [@topic_slug],
       fields: %w(title link public_timestamp format),
     }
```

Comment:
Can we not mix hash styles within one hash please. (Another reason the fat-arrow syntax is superior - you avoid this sort of thing)